src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -63,48 +67,22 @@ transpiration (Prescribed or Diagnostic)
 """
 abstract type AbstractTranspiration{FT <: AbstractFloat} end
 
-
 """
-    PlantHydraulicsParameters{FT <: AbstractFloat}
+   PrescribedSiteAI{FT <:AbstractFloat}
```

Comment:
should we spell out AI (for clarity)? PrescribedSiteAreaIndex

Reply:
💯